### PR TITLE
Do not send empty block header to slasher in validation

### PR DESCRIPTION
**Pull request by freezlite**
**What type of PR is this?**
Bug fix

**What does this PR do? Why is it needed?**
Empty block header should not be send to slasher feed. It's needed to avoid nil values in feed.

**Which issues(s) does this PR fix?**

Fixes none of the issues

**Other notes for review**
I'm not sure that feed itself will not throw away nil value, but return seems to be in goroutine after log

### CODE REVIEW

File: beacon-chain/sync/validate_beacon_blocks.go [hunk 1]
```diff
@@ -84,6 +84,7 @@ func (s *Service) validateBeaconBlockPubSub(ctx context.Context, pid peer.ID, ms
 			blockHeader, err := interfaces.SignedBeaconBlockHeaderFromBlockInterface(blk)
 			if err != nil {
 				log.WithError(err).WithField("blockSlot", blk.Block().Slot()).Warn("Could not extract block header")
+				return
 			}
 			s.cfg.slasherBlockHeadersFeed.Send(blockHeader)
```
Comment (terencechain) [inline beacon-chain/sync/validate_beacon_blocks.go:89]: Not sure about returning and skipping the rest of the validation. How about an else?

```suggestion
			} else {
			s.cfg.slasherBlockHeadersFeed.Send(blockHeader)
			}
```

Comment (freezlite) [reply beacon-chain/sync/validate_beacon_blocks.go:89]: It's incapsulated in function call, so return seems fine here, it only skip sending empty block header to feed

Comment (terencechain) [reply beacon-chain/sync/validate_beacon_blocks.go:89]: Missed that. You are right!

**Review by terencechain — APPROVED** [review]
Thanks for this!

**Comment by rauljordan** [comment]
hi @freezlite can you sign the contributor's license agreement found in comments above?

**Comment by freezlite** [comment]
@rauljordan done

**Pull request merged**